### PR TITLE
[release-0.12] Update references to release v0.12.0

### PR DESCRIPTION
**Pull request by marquiz**
Generated with:

  hack/prepare-release.sh -g 1.19.4 v0.12.0

### CODE REVIEW

**Comment by marquiz** [comment]
/retitle [release-0.12] Update references to release v0.12.0

File: docs/_config.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ plugins:
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.12
 display_version_list: true
```
Comment (fmuyassarov) [inline docs/_config.yml:47]: so we don't have to mention it like v0.12.0 right?

**Review by fmuyassarov — COMMENTED** [review]
Looks good 🎉
Just curios, why the `imagePullPolicy` is also being changed?

**Comment by fmuyassarov** [comment]
/lgtm
/hold
@marquiz feel free to unhold if you feel above questions are not something to be fixed.

File: docs/_config.yml (hunk 1 above)
Comment (ArangoGutierrez) [reply docs/_config.yml:47]: yup

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

File: docs/_config.yml (hunk 1 above)
Comment (fmuyassarov) [reply docs/_config.yml:47]: thanks @ArangoGutierrez . removing the hold
/hold cancel

**Pull request merged**